backend/src/main/java/com/skr/virtuallibrary/controllers/UserController.java

```diff
@@ -34,4 +39,17 @@ public UserDto changeLanguage(@Parameter @PathVariable String id, @Valid @Reques
     public UserDto getCurrentUser() {
         return modelMapper.toUserDto(userService.getCurrentUser());
     }
+
+    @PutMapping
+    @Operation(summary = "Update user.")
+    public UserDto updateUser(
+            @Valid @RequestPart("user")UpdateUserRequest request,
```

Comment:
Tutaj brakuje spacji
```suggestion
            @Valid @RequestPart("user") UpdateUserRequest request,
```